src/components/popover/popover.tsx

```diff
@@ -62,7 +62,6 @@ import { ESCAPE } from '../../util/keycodes';
  *
  * @slot - Content to put inside the surface
  * @exampleComponent limel-example-popover
- * @private
```

Comment:
Even though this technically only affects the docs, I think the commit should be a `feat` since, while we are _technically_ only making the docs public, and the feature was public all along, the idea is of course that people should not have been using the feature, but now they can, so we are "adding" a feature 🙂

Reply:
I agree, fixed! 🙂